gcs-action/.github/workflows/test.yml

```diff
@@ -0,0 +1,71 @@
+name: 'build-test'
+on: # rebuild any PRs and main branch changes
+  pull_request:
+  push:
+    branches:
+      - master
+      - 'releases/*'
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: GoogleCloudPlatform/github-actions/setup-gcloud@master
+        with:
+          version: '281.0.0'
+          service_account_email: ${{ secrets.GCP_SA_EMAIL }}
+          service_account_key: ${{ secrets.GCP_SA_KEY }}
+          export_default_credentials: true
+      - run: |
+          npm install
+          npm run all
+  test-upload-folder:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: GoogleCloudPlatform/github-actions/setup-gcloud@master
+        with:
+          version: '281.0.0'
+          service_account_email: ${{ secrets.GCP_SA_EMAIL }}
+          service_account_key: ${{ secrets.GCP_SA_KEY }}
+          export_default_credentials: true
+      - name: make temp folder and files
+        run: |
+          mkdir -p test
+          touch test/test1.txt
+          touch test/test2.txt
+      - uses: ./
+        with:
+          service_account_key: ${{ secrets.GCP_SA_KEY }}
+          folder-to-upload: ./test
+          bucket-name: gh-test-tf-1
+          object-key-prefix: myfolderprefix/
+          clear-existing-files-first: true
+        id: test_op
+      - name: get output
+        run: echo '${{ steps.test_op.outputs.uploadedSuccessFiles }}'
+  test-upload-file:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: GoogleCloudPlatform/github-actions/setup-gcloud@master
+        with:
+          version: '281.0.0'
```

Comment:
No need to specify version, the latest version is the default.

Reply:
Done